graylog2-server/src/main/java/org/graylog2/configuration/ElasticsearchClientConfiguration.java

```diff
@@ -45,4 +47,46 @@
 
     @Parameter(value = "elasticsearch_max_total_connections_per_route", validators = { PositiveIntegerValidator.class })
     private int elasticsearchMaxTotalConnectionsPerRoute = 2;
+
+    @Parameter(value = "elasticsearch_version")
+    private int elasticsearchVersion = 5;
+
+    public List<URI> getHosts() {
+        return elasticsearchHosts;
+    }
+
+    public Duration getConnectTimeout() {
+        return elasticsearchConnectTimeout;
+    }
+
+    public Duration getSocketTimeout() {
+        return elasticsearchSocketTimeout;
+    }
+
+    public Duration getIdleTimeout() {
+        return elasticsearchIdleTimeout;
+    }
+
+    public int getMaxTotalConnections() {
+        return elasticsearchMaxTotalConnections;
+    }
+
+    public int getMaxTotalConnectionsPerRoute() {
+        return elasticsearchMaxTotalConnectionsPerRoute;
+    }
```

Comment:
Why do we need all those getters when they are not being used anywhere? Currently we use named injection in all places, so these getters are unnecessary. We can introduce them when we really use them.